src/services/panel.ts

```diff
@@ -333,3 +336,5 @@ export function getQueryRunnerFromProvider(provider: SceneDataProvider): SceneQu
 
   throw new Error('SceneDataProvider is missing SceneQueryRunner');
 }
+
+export const logsControlsSupported = config.buildInfo.version > '12' || config.buildInfo.version.includes('12.1');
```

Comment:
I was just about to ask if we need to wait until 12.1 is released, but it looks like we'll support both with/without for a bit. Should we create an issue when <12.1 is deprecated to remove this and clean up the old code paths?

Reply:
And should this be moved to guards, or somewhere else outside of the panel.ts?

Reply:
Issue created. About placement, I couldn't find a great fit for this. The rationale here was that this is related with the underlying panel version :/

Reply:
Works for me.